rasa_nlu/classifiers/keyword_intent_classifier.py

```diff
@@ -0,0 +1,40 @@
+from rasa_nlu.components import Component
+
+
+class KeywordIntentClassifier(Component):
+
+    name = "intent_classifier_keyword"
```

Comment:
They get executed on `import` so basically, as soon as the class enters the import cache